build/SignToolData.json

```diff
@@ -4,15 +4,133 @@
             "certificate": "Microsoft402",
             "strongName": "MsSharedLib72",
             "values": [
-                "bin/Sdks/Microsoft.NET.Sdk/tools/netcoreapp2.0/Microsoft.NET.Build.Tasks.dll",
-                "bin/Sdks/Microsoft.NET.Sdk/tools/netcoreapp2.0/*/Microsoft.NET.Build.Tasks.resources.dll",
-                "bin/Sdks/Microsoft.NET.Sdk/tools/net46/Microsoft.NET.Build.Tasks.dll",
-                "bin/Sdks/Microsoft.NET.Sdk/tools/net46/*/Microsoft.NET.Build.Tasks.resources.dll",
-                "bin/Sdks/Microsoft.NET.Build.Extensions/msbuildExtensions/Microsoft/Microsoft.NET.Build.Extensions/tools/netcoreapp2.0/Microsoft.NET.Build.Extensions.Tasks.dll",
-                "bin/Sdks/Microsoft.NET.Build.Extensions/msbuildExtensions/Microsoft/Microsoft.NET.Build.Extensions/tools/netcoreapp2.0/*/Microsoft.NET.Build.Extensions.Tasks.resources.dll",
-                "bin/Sdks/Microsoft.NET.Build.Extensions/msbuildExtensions/Microsoft/Microsoft.NET.Build.Extensions/tools/net46/Microsoft.NET.Build.Extensions.Tasks.dll",
-                "bin/Sdks/Microsoft.NET.Build.Extensions/msbuildExtensions/Microsoft/Microsoft.NET.Build.Extensions/tools/net46/*/Microsoft.NET.Build.Extensions.Tasks.resources.dll",
+                "bin/Sdks/Microsoft.NET.Sdk/tools/*/Microsoft.NET.Build.Tasks.dll",
+                "bin/Sdks/Microsoft.NET.Sdk/tools/*/*/Microsoft.NET.Build.Tasks.resources.dll",
+                "bin/Sdks/Microsoft.NET.Build.Extensions/msbuildExtensions/Microsoft/Microsoft.NET.Build.Extensions/tools/*/Microsoft.NET.Build.Extensions.Tasks.dll",
+                "bin/Sdks/Microsoft.NET.Build.Extensions/msbuildExtensions/Microsoft/Microsoft.NET.Build.Extensions/tools/*/*/Microsoft.NET.Build.Extensions.Tasks.resources.dll"
+            ]
+        },
+        {
+            "certificate": null,
+            "strongName": null,
+            "values": [
+                "packages/*.nupkg"
             ]
         }
+    ],
+    "exclude": [
+        "Microsoft.DotNet.PlatformAbstractions.dll",
+        "Microsoft.Extensions.DependencyModel.dll",
+        "Microsoft.Win32.Primitives.dll",
+        "netfx.force.conflicts.dll",
```

Comment:
I'm surprised that most of these are in our nupkg. They are supposed to be pulled in separately by the CLI from the corefx support package.

Reply:
I believe most of them are in the `tools` directories.

Reply:
(for netcoreapp)

Reply:
Most of them are from Microsoft.NET.Build.Extensions\msbuildExtensions\Microsoft\Microsoft.NET.Build.Extensions, but those are excluded from the CLI  ingestion here: https://github.com/dotnet/cli/blob/5c35438cfe1ac17fd4c748b8be6c2818d1238807/build/MSBuildExtensions.targets#L10-L17 

We could (and probably should) exclude them from the .nupkg and not need to update these exclusions when the corefx support package changes.

Reply:
tools\netcoreapp2.0 has nothing but our assembly + satellites. Some of these are from tools/net46 and necessary, but not all of them.

Reply:
Fair enough, but that looks like it should be a separate work item, since this has been like this since before the infrastructure switch.

Reply:
Agreed. I filed https://github.com/dotnet/sdk/issues/1865